### DISCUSSION TITLE
Sorting As Interger

### DISCUSSION THREAD

**Discussion by curkle** [Smile-SA/elasticsuite#2505] [Ideas]
We have some text field but input is integer or float, is there anyway to sorting like number?
e.g.  option  for each products are 1,2,10,12 

expect result 
1
2
10
12 

actually result 
1
10
2
12

since it sorting as char.

Thanks.

**Comment by amenk** [comment]
You have the following sort options in the admin panel, attribute, storefront properties:

![image](https://user-images.githubusercontent.com/1087128/171598491-728435fa-26f0-4cde-b215-87373dfb89b1.png)

It does not support a "natural sort" by default, where the integers would be sorted properly.

If you have a dropdown attribute, you can ensure that the values are properly sorted in the admin panel of the attribute (Manage labels) and use the Facet sort order option "Admin sort"
